README.md

```diff
@@ -33,7 +33,7 @@ This is a monthly meeting, where the management tells all the employees how the
 
 
 # Your First Day
-- Meet with the [Head of iOS](https://nodes.slack.com/messages/@chco), your [manager](https://nodes.slack.com/messages/@mofa) and your Nodes Buddy.
+
```

Comment:
I don't think we need to remove this, I just think we shouldn't specify the people :)

Reply:
> I've removed a line related to onboarding which I think was very specific to iOS devs in Copenhagen; I'm not familiar with `halp`, I don't think I was introduced to it yet 🙈, so maybe I'm not the best person to write about it.

Halp is just a slack command `/halp` for getting help from IT

Reply:
Yeah, I had removed it because I wasn't sure how onboarding for iOS devs in other offices goes. Do they have an online meeting with the head of ios?

Reply:
Yes, that's how it works everywhere as far as I know.